functions/src/db/updateInvitationSentCounts.ts

```diff
@@ -0,0 +1,21 @@
+import { db, increment, serverTimestamp } from '../firebase/firebase'
+import { COLLECTION_USERS } from './constants'
+import { UID } from '../types/uid'
+
+export const updateInvitationSentCounts = async (
+    userId: UID,
+    smssSentCount: number,
+    emailsSentCount: number
+) => {
+    return await db
```

Comment:
same as in the other PR: `await` is not needed here

Reply:
I added a rule [here](https://github.com/Instant-Visio/InstantVisio-WebApp/pull/306) not to forget to fix this one